File: Backend/mapping/src/main/java/com/mhp/coding/challenges/mapping/models/dto/ArticleDto.java
package com.mhp.coding.challenges.mapping.models.dto;


import com.mhp.coding.challenges.mapping.models.dto.blocks.ArticleBlockDto;

import java.util.Collection;

/**
 * @author Asdren Hoxha (MHP) - 17.09.18.
 */
public class ArticleDto {

    private String title;

    private String description;

    private String author;

    private Collection<ArticleBlockDto> blocks;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getAuthor() {
        return author;
    }

    public void setAuthor(String author) {
        this.author = author;
    }

    public Collection<ArticleBlockDto> getBlocks() {
        return blocks;
    }

    public void setBlocks(Collection<ArticleBlockDto> blocks) {
        this.blocks = blocks;
    }
}

File: Backend/mapping-kotlin/src/main/kotlin/com/mhp/coding/challenges/mapping/models/dto/ArticleDto.kt
package com.mhp.coding.challenges.mapping.models.dto

import com.mhp.coding.challenges.mapping.models.dto.blocks.ArticleBlockDto

class ArticleDto(
    var id: Long? = null,
    var title: String? = null,
    var description: String? = null,
    var author: String? = null,
    var blocks: Collection<ArticleBlockDto>? = null,
)
